tests/DfE.FindInformationAcademiesTrusts.UnitTests/Pages/OtherServicesLinkBuilderTests.cs

```diff
@@ -106,4 +106,21 @@ public void FindSchoolPerformanceDataListingLink_should_be_null_if_trust_is_sing
         });
         result.Should().BeNull();
     }
+    
+    [Fact]
+    public void SharepointFolderLink_should_return_url_containing_CompaniesHouseNumber()
+    {
+        var result =
+            _sut.SharepointFolderLink(DummyTrustDetailsServiceModel with { GroupId = "2345" });
```

Comment:
Would be good to use a value that looks more like a TRN here to highlight which ID number we're using